doc/sphinx/source/tutorials/futuretests.rst

```diff
@@ -0,0 +1,83 @@
+.. _futuretests:
+How to run a Future Test
+========================
+
+The PDF sets generated with the NNPDF methodology are, of course, able to describe the data they were fitted with.
+However, for them to be truly useful, they should be also able to faithfully describe data that the methodology has never seen.
+By checking the :math:`\chi2` of said unseen data we can test the generalization power of the methodology.
+
+Ideally, that would entail constructing a new experiment and obtaining new data we can test the PDF against,
+however, we cannot time travel so we must resort to less robust techniques.
+In NNPDF4 we test the generalization power of the methodology by fitting coherent subsets of data.
+In order to keep consistent with the analogy, in :cite:p:`Cruz-Martinez:2021rgy` we chose Pre-Hera and Pre-LHC as
+those subsets but in reality (and despite the name) any subset of data would be valid.
+The chronological choice, which in practice separates by experiment, ensures that no leakage of information occurs between the in-sample and out-of-sample data
+(for instance, some unknown systematic error that biases data from one given experiment).
+
+Since we are interested in the generalization power of the PDF, we need to take into account the pdf errors when
+computing the :math:`\chi2`, to that end the flag ``use_pdferr`` is provided in ``validphys``.
```

Comment:
Maybe give the explicit expression of chi2 if use_pdferr is true? It's quite easy to understand and removes any ambiguity of what "taking into account the pdf errors" actually means.

Reply:
You're right. I'll add it